app/response/utils.py

```diff
@@ -424,6 +456,29 @@ def process_email_template_request(request_id, data):
                                agency_name=agency_name,
                                note_content=note_content,
                                page=page)
+    # process email template for offline instructions
+    if data['type'] == 'instruction_email':
```

Comment:
Python does not have a switch statement, which would be great here. Instead you can use a dictionary of email types (**which should be constants by the way!**) to handler functions like this: 
```
handler_for_type = {
    'note_email': handle_note_email,
    'instruction_email': handle_instruction_email,
    ...
}
handler_for_type[data['type'])(...)
```